components/html-block/html-block.js

```diff
@@ -1,6 +1,6 @@
 import '../colors/colors.js';
 import { css, LitElement } from 'lit-element/lit-element.js';
-import { HtmlAttributeObserverController } from '../../helpers/htmlAttributeObserverController.js';
+import { HtmlAttributeObserverController } from '../../controllers/attributeObserver/htmlAttributeObserverController.js';
```

Comment:
`attributeObserver` is fairly generic, but there were some suggestions that we could extend this controller (or create a new one) to handle the more generic case of observing attributes on an arbitrary element. As such, I figure it might be good to leave room for that.